### ISSUE TITLE
Screen readers do not announce the role for the column headers when the table is empty using 'Ctrl + Alt+Arrow' key in windows.

### ISSUE THREAD

**Issue by msftedad** [ceolter/ag-grid#13821]
### Link to reproducible scenario

https://www.ag-grid.com/example-inventory/

### Describe the bug

**Test Environment:**
OS: Windows 11 Version 25H2 (OS Build 26200.8390)
Browsers: Edge (Version 148.0.3967.70), Chrome (Version 148.0.7778.168)
Screen Reader: NVDA (2026.1), JAWS (2026.2603.86), Narrator
URL: [Demo - Inventory Management](https://www.ag-grid.com/example-inventory/)

**User Impact:** 
Screen reader users will be impacted as column header roles are not announced when navigating an empty table. This can make it difficult for users to understand the structure and context of the table, leading to confusion while navigating.

**Pre-Requisite:** 
Turn on Screen Readers:
NVDA: Ctrl + Alt + N
JAWS: Ctrl + Alt + J
Narrator: Ctrl + win + Enter 
Verbosity:
NVDA: Default
JAWS: Beginner, Highest 
Narrator: Default

**Repro Steps:**

1. Open the URL: [Demo - Inventory Management](https://www.ag-grid.com/example-inventory/) in edge/chrome browser.
2. 'Demo - Inventory Management' page will open.
3. Press tab key to reach to search edit and any random data.
4. Empty table will appear.
5. Press 'T' and then 'Ctrl + Alt + arrow' key.
6. Verify whether screen readers announce the role for the column headers when the table is empty using 'Ctrl + Alt+Arrow' key in windows or not.

**Actual Result:**
Screen readers do not announce the role for the column headers when the table is empty using 'Ctrl + Alt+Arrow' key in windows.
Screen reader announces: Column 1 clickable album name

**Refer Attachment:**

1. Screen readers do not announce the role for the column headers when the table is empty using 'Ctrl + Alt+Arrow' key in windows.png
2. Screen readers do not announce the role for the column headers when the table is empty using 'Ctrl + Alt+Arrow' key in windows.mp4

https://github.com/user-attachments/assets/7eb1c128-21b5-4862-8a4b-55c26aeadfd8
<img width="1949" height="1214" alt="Image" src="https://github.com/user-attachments/assets/213185bf-b1e0-4686-aec7-26d7590f5228" />

**Expected Result:**
Screen readers should announce the role for the column headers when the table is empty using 'Ctrl + Alt+Arrow' key in windows.
Screen reader should announce: Column 1 clickable column header album name

### Version

v35.3.0

### Does the issue occur for a specific framework only?

All languages

### Is the issue only observable on a specific browser?

All browsers
